cmd/up/activate.go

```diff
@@ -382,7 +382,7 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context,
 			// this provides 2 min for "FailedScheduling" to resolve by themselves
 			if strings.Contains(failedSchedulingEvent.Message, "Insufficient cpu") || strings.Contains(failedSchedulingEvent.Message, "Insufficient memory") {
 				return oktetoErrors.UserError{E: fmt.Errorf("insufficient resources"),
-					Hint: "Increase cluster resources or timeout of resources. More information is available here: https://okteto.com/docs/reference/manifest/#timeout-time-optional"}
+					Hint: "Increase context resources or timeout of resources. More information is available here: https://okteto.com/docs/reference/manifest/#timeout-time-optional"}
```

Comment:
```suggestion
					Hint: "Increase cluster resources or timeout of resources. More information is available here: https://okteto.com/docs/reference/manifest/#timeout-time-optional"}
```
I thing in this case is clearer talk about cluster